providers/implementations/serializers/deserialize_der2rsa.c

```diff
@@ -37,10 +39,12 @@ static OSSL_FUNC_deserializer_export_object_fn der2rsa_export_object;
 struct der2rsa_ctx_st {
     PROV_CTX *provctx;
 
+    int type;
+
     struct pkcs8_encrypt_ctx_st sc;
 };
 
-static void *der2rsa_newctx(void *provctx)
+static void *der2rsa_newctx_int(void *provctx)
```

Comment:
I would have passed the type in here also.. but this is ok.

Reply:
Done